map_view/display-device-location-with-autopan-modes/README.metadata.json

```
@@ -0,0 +1,37 @@
+{
+    "category": "Map",
+    "description": "Display your current position on the map, as well as switch between different types of auto pan modes.",
+    "ignore": false,
+    "images": [
+        "DisplayDeviceLocation.png"
```

ok maybe looks like this metadata file hasn't been updated since last review changes? seems to be a few outdated things including the .png. I'll re-review this once it's gone through the script again :) 
```suggestion
        "DisplayDeviceLocationWithAutopanModes.png"
```

---

Yes, I forgot to run the script again after the rename!